www/md_base/src/app/index.jade

```diff
@@ -0,0 +1,45 @@
+doctype html
+html(ng-app="app", ng-controller="appController as app")
```

Comment:
I am happy to see the controller as syntax. Don't forget to change the $scope.title to @title in the appController and to app.title in the view.